Practice2/src/partials/features-section.html

```diff
@@ -0,0 +1,47 @@
+<section class="page-features">
+  <h1 class="features-heading">Features</h1>
+  <h4 class="features-heading-sub">We're talking it to the next level</h4>
```

Comment:
How can the heading jump from h1 to h4 suddenly like this?

Reply:
Because it's a subtitle

Reply:
Because it's a subtitle

Reply:
@duylong13 
1. We have different levels of heading. Why don't we use `h2`, the next lower level of `h1` here instead of `h4`?
2. In addition, `h1` is for the most important heading (the highest top level of heading) of a site. Is it suitable to be used here?

Reply:
@huynguyen-agilityio 
1. I changed back <h4> to <h2>
2. That's the beginning of a section, so I'll use <h1>